k2/torch/csrc/deserialization.h

```diff
@@ -22,9 +22,27 @@
 #include <string>
 
 #include "k2/csrc/fsa.h"
+#include "k2/torch/csrc/utils.h"
 
 namespace k2 {
 
+void RegisterRaggedInt();
```

Comment:
This function and the following class `RaggedIntHelper` can be moved to `.cu` file.

Users don't need to know about them.

Reply:
Already, I will commit later.